packages/toml/toml.3.0.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+
+available: [ ocaml-version >= "4.01.0" ]
+
+version: "3.0.0"
+
+authors: [ "Julien Sagot" "Emmanuel Surleau" "mackwic" ]
+
+maintainer: "support@toml.epimeros.org"
+
+homepage: "http://mackwic.github.io/To.ml/"
+
+build: [ [ "./configure"  "--prefix=%{prefix}%" ]
+         [ make "build" ] ]
+
+install: [ [ make "install" ] ]
+
+remove: [ "ocamlfind" "remove" "toml" ]
+
+depends: [ "ocamlfind" "menhir" "ISO8601" ]
+
+bug-reports: "https://github.com/mackwic/To.ml/issues"
+
+dev-repo: "https://github.com/mackwic/To.ml/"
```

Comment:
This needs to end in `.git` instead of `/` in order for opam to recognize the version control protocol.